docs/fastdds/discovery/discovery_server.rst

```diff
@@ -79,9 +79,17 @@ In this architecture there are several key concepts to understand:
     since several *servers* can be hosted in the same machine, thus having the same IP, and also because multicast
     addresses are acceptable addresses.
 
-- A |SUPER_CLIENT| is a *client* that receives all the discovery information known by the *server*, in opposition to
+- A |SUPER_CLIENT| is a *client* that receives the discovery information known by the *server*, in opposition to
   *clients*, which only receive the information they need.
 
+  .. note::
+
+    A *Super_client* does not behave as a *Server* as it only receives the discovery information through the *Server* it
+    is connected to.
```

Comment:
Mind the line length
```suggestion
    A |SUPER_CLIENT| does not behave as a *Server* as it only receives the discovery information through the *Server* to which it is connected.
```

Reply:
It is just in the line length limit :laughing: The CI has passed without complaining :wink: